lib/administrate/field/rich_text.rb

```diff
@@ -0,0 +1,11 @@
+require "administrate/field/base"
+
+module Administrate
+  module Field
+    class RichText < Administrate::Field::Base
```

Comment:
Would we get _any_ benefit from inheriting from the plain text plugin here, or would that just add more complexity for no benefit?